### PR TITLE
Set sample allow empty string

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds the possibility to set empty capture kit via CLI

**How to prepare for test**:
- [x] ssh to hasta
- [x] install on stage:
`bash servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

**How to test**:
- [x] login to hasta
- [x] do cg set sample [sample] -k "a_capture_kit"
- [x] do cg set sample [sample] -k ""

**Expected test outcome**:
- [x] check that cg set the capture kit to empty
- [x] Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by PG
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: cg/cli/set.py [hunk 1]
```diff
@@ -230,25 +200,20 @@ def sample(
...
-                        fg="green",
+                        f"\nApplication tag set to {str(application_version)}.", fg="green"
                     )
 
         if not skip_lims:
             context.obj["lims"].update_sample(sample_id, application=apptag)
             click.echo(click.style(f"Set LIMS/application to {apptag}", fg="blue"))
 
-    if capture_kit:
+    if capture_kit is not None:
```
Comment (northwestwitch) [inline cg/cli/set.py]: This is the only change right? 👍

Comment (northwestwitch) [reply cg/cli/set.py]: Better this way, I saw many errors due to the previous code!

**Comment by patrikgrenfeldt** [comment]
Test result:
image
👍

File: cg/cli/set.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/cli/set.py]: right

**Comment by patrikgrenfeldt** [comment]
deployed on hasta @annaengstrom

**Comment by annaengstrom** [comment]
Wonderful! What does `--skip-lims` do?

Also, is there anywhere we keep documentation or these is more information for these commands? For example how they change LIMS and what the comment 3 digit code in statusdb means?

**Pull request merged**